packages/react/src/componentsV2/addressSection/index.tsx

```diff
@@ -0,0 +1,179 @@
+import { SearchIcon } from '@heliofi/helio-icons';
+import { useSelect } from 'downshift';
+import { useEffect } from 'react';
+import { useAddressProvider } from '../../providers/address/AddressContext';
+import { Button, Input } from '../../ui-kits';
+
+import {
+  StyledAreaCode,
+  StyledButton,
+  StyledInput,
+  StyledSelectDropdown,
+  StyledSelectDropdownContainer,
+  StyledSelectItem,
+  StyledWrapper,
+  StyledStreetInfo,
+  StyledStreet,
+  StyledStreetNumber,
+} from './styles';
+
+type AddressSectionProps = {
+  values: {
+    deliveryAddress?: string;
+    city?: string;
+    streetNumber?: string;
+    street?: string;
+  };
+  setFieldValue: (field: string, value: string | number) => void;
```

Comment:
You declare this type at least 3 times in the PR, maybe better to make a type for it and import